docs/nodes/providers/2_setting_up.md

```diff
@@ -40,24 +40,41 @@ Your keybase identity can be left as `""` if you don't want to link your provide
 
 :::
 
-:::note
-
-Before running 'jprovd init', your account will need tokens. 
-Testnet tokens can be obtained at https://testnet-faucet.jackalprotocol.com/ 
-Mainnet tokens can be purchased on https://frontier.osmosis.zone/ 
-
-:::
-
 ```sh
-export PHOME={path-to-pool-from-before}
-
-jprovd client gen-key --home=$PHOME
-
-jprovd client config chain-id {chain-id} --home=$PHOME
-jprovd client config node {your node} --home=$PHOME
-
-jprovd init {IP_ADDRESS} {STORAGE_IN_BYTES} {KEYBASE_IDENTITY} --home=$PHOME
-
-jprovd start --home=$PHOME
+1. export PHOME={path-to-pool-from-before}
+
+2. jprovd client gen-key --home=$PHOME
+
+  Before completing step 5 below, your account will need tokens. 
+  Testnet tokens can be obtained at https://testnet-faucet.jackalprotocol.com/ 
+  Mainnet tokens can be purchased on https://frontier.osmosis.zone/ 
+
+3. jprovd client config chain-id {chain-id} --home=$PHOME
+4. jprovd client config node {your node} --home=$PHOME
+
+  Some users may find it easier to complete steps 3 and 4 manually. Instructions for Ubuntu are below:
+  
+  a. In your terminal, while inside of the PHOME directory, type 'nautilus'. This will open the nautilus file explorer
+  b. At the top right corner, make sure 'Show Hidden Files' is checked
+  c. Navigate to your '.jackal-storage' folder to access the config folder
+  d. Open the client.toml file to manually type in the 'chain-id' and 'node'
+  e. Save and close the client.toml file
+  
+  Testnet and Mainnet chain-ids can be found here: https://github.com/JackalLabs/jackal-chain-assets.
+  If you are not running your own tendermint RPC node, Jackal Labs is currently hosting a testnet tendermint RPC node. Endpoint below:
+  
+  https://testnet-rpc.jackalprotocol.com:443
+  
+  We will also be hosting a tendermint rpc node for mainnet, which will be made accessible to the public at launch of Jackal Storage. 
+  
+5. jprovd init {IP_ADDRESS} {STORAGE_IN_BYTES} {KEYBASE_IDENTITY} --home=$PHOME
```

Comment:
steps like this should be defined as:

5. Initialize the provider
```sh
jprovd init {IP_ADDRESS} {STORAGE_IN_BYTES} {KEYBASE_IDENTITY} --home=$PHOME
```

Reply:
I've revised the documents accordingly.